src/api/status/Dockerfile

```diff
@@ -9,6 +9,8 @@ COPY --chown=node:node . .
 
 RUN npm install --only=production --no-package-lock
 
+RUN npm run compile:scss
```

Comment:
This is not going to work since we're using `node:lts-alpine` as our base image (`lts` is 16).
I'd suggest changing the first line of this file to:

```yml
FROM node:14.18-alpine as base
```
I tested this and docker can build and launch it.

Reply:
@Andrewnt219 does this still apply with your latest change to sass vs. node-sass?  If it's not a problem any more, please resolve this.

Reply:
@manekenpix do we still need this line when the node engine is no longer restricted to 14 and below?

Reply:
I don't think so. Your latest changes seem to work with node v16, so I think we're ok.